ironfish-cli/src/commands/airdrop/airdrop.ts

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { CliUx } from '@oclif/core'
+import fs from 'fs/promises'
+import { IronfishCommand } from '../../command'
+import { watchTransaction } from '../../utils/transaction'
+
+export class Airdrop extends IronfishCommand {
+  static description = `Drop coins to testnet participants`
+  static aliases = ['airdrop:airdrop']
+  static hidden = true
+  static args = [
+    {
+      name: 'transactionsFile',
+      parse: (input: string): Promise<string> => Promise.resolve(input.trim()),
+      description: 'New line separated serialized airdrop transactions',
+    },
+  ]
+  async start(): Promise<void> {
+    const { args } = await this.parse(Airdrop)
+    let lineNum = 0
+    const fileContent = await fs.readFile(args.transactionsFile, 'utf-8')
+    const lines = fileContent.split(/[\r\n]+/)
+    const client = await this.sdk.connectRpc()
+    for (const line of lines) {
+      lineNum++
+      CliUx.ux.action.start(`Adding transaction #${lineNum}`)
+      const response = await client.addTransaction({ transaction: line.trim() })
+      CliUx.ux.action.stop()
+      if (response.content.accepted) {
+        this.logger.info(`Added ${response.content.hash} transaction (#${lineNum})`)
+        await watchTransaction({
+          client,
+          logger: this.logger,
+          account: response.content.accounts[0],
+          hash: response.content.hash,
+        })
+      } else {
+        this.logger.warn(
+          `Skipping ${response.content.hash} transaction (#${lineNum}), transaction was not accepted`,
+        )
+      }
+    }
+  }
+}
```

Comment:
can we move this into the service commands?

Reply:
I don't think it matters where we put this stuff, we are never going to merge it into staging/master (unless you think we should).

I have a root branch `airdrop` that I was planning on using for the actual airdrop.